.travis.yml

```diff
@@ -1,57 +1,56 @@
 dist: xenial
 language: python
-python: 3.6.8
-
-sudo: false
-
-# The apt packages below are needed for sphinx builds
-addons:
-  apt:
-    packages:
-      - texlive-latex-extra
-      - dvipng
-      - graphviz
-      - libopenblas-base
-      - libopenblas-dev
-
-env:
-  global:
-    - NUMPY_VERSION=1.16
-    - DEV="--upgrade cython git+https://github.com/numpy/numpy git+https://github.com/astropy/astropy"
-
-matrix:
+python: 3.8.5
+
+jobs:
   # Don't wait for allowed failures
   fast_finish: true
   include:
-    # Run tests
-    - env: TEST_COMMAND='pytest'
-           PIP_DEPENDENCIES='-e .[test]'
-    # Build sphinx documentation with warnings
-    - env: TEST_COMMAND='python setup.py build_sphinx'
-           PIP_DEPENDENCIES='-e .[docs]'
-    # Test with python 3.7
-    - python: 3.7
-      env: TEST_COMMAND='pytest'
-           PIP_DEPENDENCIES='-e .[test]'
-    # Cron job test with dev versions of dependencies and python 3.6
-    - env: PIP_DEPENDENCIES="${DEV} -e .[test]"
-           TEST_COMMAND='pytest'
-           # EVENT_TYPE='cron'
-    # Cron job test with dev versions of dependencies and python 3.7
-    - python: 3.7
-      env: PIP_DEPENDENCIES="${DEV} -e .[test]"
-           TEST_COMMAND='pytest'
-           # EVENT_TYPE='cron'
-    # PEP8 check
-    # Strict PEP8 check, an allowed failure below
-    - env: TEST_COMMAND='flake8 --count'
+    - name: Latest dependencies
+
+    - name: Old dependencies
+      python: 3.6.10
+      install:
+        - pip install numpy~=1.16.0
+        - "pip install -e .[test]"
+
+    - name: Documentation build
+      install:
+        - "pip install -e .[docs]"
+      script:
+        - sphinx-build docs docs/_build
+      addons:
+        apt:
+          packages:
+            - libopenblas-base
+            - libopenblas-dev
+
+    - name: Dev dependencies
```

Comment:
Might want to do this one as cron job but to now allow to fail, up to you.

In my experience, allowing something to fail cause people not to notice until it is too late, especially if the developer is often busy with something else.

Reply:
Agree.  I will remove the allowed failure, but keep it on a per-PR for now.  I don't want to have to fiddle with cron settings in the Travis setup right now.